zammad/templates/configmap-init.yaml

```diff
@@ -56,6 +56,10 @@ data:
     bundle exec rake zammad:searchindex:rebuild
   {{ else }}
     echo "Checking if an elasticsearch index already exists…"
+
+    # Ensure ES connectivity, as SearchIndexBackend.index_exists? swallows internal errors.
+    bundle exec rails r "SearchIndexBackend.version"
+
     if bundle exec rails r "SearchIndexBackend.index_exists?('Ticket') || exit(1)"
```

Comment:
is `|| exit(1)` really needed if `set - e` is used above?

Reply:
Yes, it is in the Ruby context. search_index_exists just returns true or false in Ruby, but we want it to return with non-zero exit code in case it is false, hence the exit(1) (also in Ruby context).